### PR TITLE
implement a system that makes it possible to set matrixes as not console operable

### PR DESCRIPTION
**Pull request by AutumnNova**
maybe

### CODE REVIEW

**Comment by corp-0** [comment]
do all matrixes have the MatrixMove component? Wouldn't not adding the component be enough to mark a matrix as not piloteable?

**Comment by AutumnNova** [comment]
as far as i know things like the cargo shuttle and ems need that component to function

**Comment by corp-0** [comment]
ah, that makes sense

File: UnityProject/Assets/Scripts/Shuttles/ShuttleConsole.cs [hunk 1]
```diff
@@ -72,7 +72,11 @@ private IEnumerator InitMatrixMove()
 					Category.Matrix);
 			}
 
-			if (ShuttleMatrixMove != null)
+			if (ShuttleMatrixMove.NoConsole == true)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Shuttles/ShuttleConsole.cs]:
```suggestion
			if (ShuttleMatrixMove.NoConsole)
```
When checking a bool, you don't really need to compare it to true.

File: UnityProject/Assets/Scripts/Shuttles/MatrixMove.cs [hunk 2]
```diff
@@ -177,6 +177,9 @@ public class MatrixMove : ManagedNetworkBehaviour
 	/// </summary>
 	public bool Initialized => clientStarted && receivedInitialState;
 
+	[Tooltip("Disable the ability for players to use a shuttleconsole to control this matrix")]
+	public bool NoConsole = false;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Shuttles/MatrixMove.cs]: Although the tooltip is great, maybe this name is not the best to understand what it does at first glance. Change it to something like ``IsPilotable``. You would need to invert the logic check if you choose that name.

**Pull request merged**